src/command/list/human.rs

```diff
@@ -58,46 +114,272 @@ fn display_all(
             .subsequent_indent(INDENTATION)
             .fill(
-                &runtimes
+                &package_managers
                     .iter()
-                    .map(|runtime| format!("v{}{}", runtime.version, runtime.source))
+                    .map(list_package_manager)
                     .collect::<Vec<String>>()
                     .join("\n"),
             );
-
-        format!("⚡️ Node runtimes in your toolchain:\n\n{}", versions)
+        format!("⚡️ Yarn versions in your toolchain:\n\n{}", versions)
     }
 }
 
-/// Format the output for `Toolchain::PackageManager`.
-fn display_package_managers(package_managers: &[PackageManager]) -> String {
-    unimplemented!()
-}
-
 /// Format a set of `Toolchain::Package`s and their associated tools.
 fn display_packages(packages: &[Package]) -> String {
-    unimplemented!()
+    if packages.is_empty() {
+        String::from(
+            "⚡️ No tools or packages installed.
+
+You can safely install packages by running `volta install <package name>`.
+See `volta help install` for details and more options.",
+        )
+    } else {
+        format!(
+            "⚡️ Package versions in your toolchain:\n\n{}",
+            list_packages(packages)
+        )
+    }
 }
 
-/// Format the output for a specific tool from a set of `Toolchain::Package`s.
+/// Format a single `Toolchain::Tool` with associated `Toolchain::Package`
+
 fn display_tool(tool: &str, host_packages: &[Package]) -> String {
-    unimplemented!()
+    if host_packages.is_empty() {
+        format!(
+            "⚡️ No tools or packages named `{}` installed.
+
+You can safely install packages by running `volta install <package name>`.
+See `volta help install` for details and more options.",
+            tool
+        )
+    } else {
+        let width = text_width().unwrap_or(0);
+        let versions = Wrapper::new(width)
+            .initial_indent(INDENTATION)
+            .subsequent_indent(INDENTATION)
+            .fill(
+                &host_packages
+                    .iter()
+                    .map(list_package)
+                    .collect::<Vec<String>>()
+                    .join("\n"),
+            );
+        format!("⚡️ Tool `{}` available from:\n\n{}", tool, versions)
+    }
+}
+
+/// Format a sets binaries inside `Toolchain::Package`s without detail information
+fn display_tool_sets(packages: &[Package]) -> String {
+    format!(
+        "{}",
+        &packages
+            .iter()
+            .map(display_tool_set)
+            .collect::<Vec<String>>()
+            .join("\n")
+    )
+}
+/// Format a set binaries inside a single `Toolchain::Package` without detail information
+fn display_tool_set(package: &Package) -> String {
+    match package {
+        Package::Default { tools, .. } | Package::Project { tools, .. } => {
+            let tools = match tools.len() {
+                0 => String::from(""),
+                _ => format!("{}", tools.join(", ")),
```

Comment:
Another spot where I believe we can use `tools.join(", ")` directly, instead of using the format